server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestClearVotingConfigExclusionsAction.java

```diff
@@ -34,7 +34,9 @@ public String getName() {
 
     @Override
     protected RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client) throws IOException {
-        ClearVotingConfigExclusionsRequest req = new ClearVotingConfigExclusionsRequest();
+        final var req = new ClearVotingConfigExclusionsRequest();
+        req.masterNodeTimeout(request.paramAsTime("master_timeout", req.masterNodeTimeout()));
+        req.setTimeout(req.masterNodeTimeout());
```

Comment:
Should we simply remove the timeout field instead and just use `masterNodeTimeout`?

Reply:
I did wonder about that but it is doing something a bit complicated that I'd rather leave alone for now.